x-pack/legacy/plugins/integrations_manager/server/packages/remove.ts

```diff
@@ -21,11 +28,24 @@ export async function removeInstallation(options: {
   await savedObjectsClient.delete(SAVED_OBJECT_TYPE, pkgkey);
 
   // Delete the installed assets
-  const deletePromises = installedObjects.map(async ({ id, type }) =>
-    savedObjectsClient.delete(type, id)
-  );
+  const deletePromises = installedObjects.map(async ({ id, type }) => {
+    if (assetUsesObjects(type as AssetType)) {
+      savedObjectsClient.delete(type, id);
+    } else if (type === AssetType.ingestPipeline) {
+      deletePipeline(callCluster, id);
+    }
+  });
   await Promise.all(deletePromises);
 
   // successful delete's in SO client return {}. return something more useful
-  return installedObjects;
+  const packageInfo = await getPackageInfo({ savedObjectsClient, pkgkey });
+
+  return packageInfo;
+}
+
+async function deletePipeline(callCluster: CallESAsCurrentUser, id: string): Promise<void> {
+  // '*' shouldn't ever appear here, but it still would delete all ingest pipelines
```

Comment:
Should we do something (log, error, etc) when we get an unexpected value?

Reply:
I think we should. I'll open an issue.